geps/gep-2648/index.md

```diff
@@ -0,0 +1,415 @@
+# GEP-2648: Direct Policy Attachment
+
+* Issue: [#2648](https://github.com/kubernetes-sigs/gateway-api/issues/2648)
+* Status: Experimental
+
+(See status definitions [here](/geps/overview/#gep-states)
+
+## TLDR
+
+Describe and specify a design pattern for a class of metaresource that can
+affect specific settings across a single target object.
+
+This is a design for a _pattern_, not an API field or new object.
+
+## Goals
+
+* Specify what common properties all Direct Attached Policies MUST have
+* Recommend design patterns for areas that cannot be mandated but that could
+  cause problems for API designers.
+
+## Non-Goals
+
+* Fully specify the entire design space for Direct Policy Attachment
+
+## Introduction
+
+GEP-713 defines two classes of Policy Attachment: Direct and Inherited.
+
+Direct Attached Policies (or Direct Policies) _only_ affect the object they are
+attached to; that is, the object specified in their `targetRef`.
+
+Note that as soon as the Policy affects more objects than the referenced object,
```

Comment:
I mentioned this in a review of the gep-713 changes, but again, shouldn't a Direct Policy be able to refer to a collection/set of objects, as long as they are all at the same hierarchical level?  E.g. a targetRef of a set of common rules or specific routes; or a struct that groups non-hierarchical objects from the same namespace.

Reply:
Originally, I thought no, as it will add another layer of complexity onto the status design (each targetRef will probably need a new status...) But I think I could be talked around on this - but I think that this dicussion needs to happen _after_ this PR merges - this has already been in progress for way too long.